lms/envs/devstack.py

```diff
@@ -358,6 +358,8 @@ def should_show_debug_toolbar(request):
     # more background on edx-enterprise.
     # Toggle this off if you don't want anything to do with enterprise in devstack.
     'ENABLE_ENTERPRISE_INTEGRATION': True,
+    # Toggle this on if you want to redirect locally to new logistration MFE
+    'ENABLE_LOGIN_MICROFRONTEND': False
```

Comment:
Can use this approach to define toggles? https://github.com/edx/edx-platform/blob/c5502057b1a15c5e597ccb4e983948408bb416bd/lms/envs/common.py#L420-L431

Reply:
Also the name 
```suggestion
    'ENABLE_LOGISTRATION_MICROFRONTEND': False
```

Reply:
@waheedahmed I will use the above mentioned approach you suggested. I didn't understand your second comment, what to do with the name `ENABLE_LOGISTRATION_MICROFRONTEND` ?

Reply:
Got it, I will change the name